spec/features/purchases_spec.rb

```diff
@@ -30,6 +33,33 @@
     expect(numrows).to be == 5
   end
 
+  scenario 'display three purchases after clicking on week button' do
```

Comment:
the scenario could be expressed more accurately as `'display purchases from the last week'` - the number is given in the `expect` assertion.